go/vt/topotools/keyspace.go

```diff
@@ -54,6 +54,9 @@ func RefreshTabletsByShard(ctx context.Context, ts *topo.Server, tmc tmclient.Ta
 	var (
 		m  sync.Mutex
 		wg sync.WaitGroup
+		// Set a relatively low timeout and the callers will know if we got a partial refresh and decide
+		// what to do if so.
+		refreshTimeout = 2 * time.Second
```

Comment:
Who are the other consumers of this RPC? Is it ok to reduce it to 2s for everybody or does it need to be parameterized?